src/Nethermind/Nethermind.Db/Databases/EthRequestsRocksDb.cs

```diff
@@ -29,7 +29,7 @@ public EthRequestsRocksDb(string basePath, IDbConfig dbConfig, ILogManager logMa
         {
         }
 
-        protected override void UpdateReadMetrics() => Metrics.EthRequestsDbReads++;
-        protected override void UpdateWriteMetrics() => Metrics.EthRequestsDbWrites++;
+        protected internal override void UpdateReadMetrics() => Metrics.EthRequestsDbReads++;
```

Comment:
why would you make it internal? if so then at least protected internal

Reply:
It is protected internal. For a ColumnDb to be able to call to ColumnsDb. As its same DB for metrics.